parser/src/main/java/log/parser/service/MatchService.java

```diff
@@ -0,0 +1,10 @@
+package log.parser.service;
+
+import java.util.ArrayList;
+
+public interface MatchService {
+	
+	Long ingestMatch(String payload);
```

Comment:
I think parseLogEntry and saveDatabase methods should be kept private. At this point there should only be one method 

`Match upload(string payload);`